app/webpack/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.GraduatedPrice.js

```diff
@@ -107,7 +107,7 @@
 
     setAmount: function (data, context) {
       const $amount = $(context).find('input.fee-amount')
-      const $priceCalculated = $(context).find('.js-fee-calculator-success > input')
+      const $priceCalculated = $(context).find('.js-fee-calculator-success input')
```

Comment:
is this change necessary ? looking at the code it seems like `input` is directly under the parent div with class `js-fee-calculator-success`. 
Hence using a child selector `>` is more appropriate as this will only target the immediate child elements and not any descendants.